### PR TITLE
LatinPadBLE: correct info.json data

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

Removes an extra key object, and corrects the layout macro reference.

cc @haierwangwei2005 (keyboard maintainer)

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/latinpadble/rules.mk [hunk 1]
```diff
@@ -1,29 +1,17 @@
...
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = caterina
 F_CPU = 8000000
 
-
 # Feather uses the caterina bootloader
 OPT_DEFS += -DBOOTLOADER_SIZE=4096 -DCATERINA_BOOTLOADER
```
Comment (fauxpark) [inline keyboards/latinpadble/rules.mk:9]:
```suggestion
```

File: keyboards/latinpadble/rules.mk [hunk 2]
```diff
@@ -1,29 +1,17 @@
...
 # Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = caterina
 F_CPU = 8000000
```
Comment (fauxpark) [inline keyboards/latinpadble/rules.mk:6]:
```suggestion
```

File: keyboards/latinpadble/rules.mk [hunk 3]
```diff
@@ -1,29 +1,17 @@
-
 # MCU name
 MCU = atmega32u4
 
 # Bootloader selection
```
Comment (fauxpark) [inline keyboards/latinpadble/rules.mk:4]:
```suggestion
# Processor frequency
F_CPU = 8000000

# Bootloader selection
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/latinpadble/rules.mk (hunk 1 above)
Comment (noroadsleft) [reply keyboards/latinpadble/rules.mk:9]: I felt like this was unnecessary, but I couldn't find where it was set in core.

Comment (fauxpark) [reply keyboards/latinpadble/rules.mk:9]: `bootloader.mk`

**Review by fauxpark — APPROVED** [review]

File: keyboards/latinpadble/rules.mk (hunk 1 above)
Comment (noroadsleft) [reply keyboards/latinpadble/rules.mk]: Ahh, it was in the root directory; that's why I didn't find it.

**Pull request merged**